__tests__/api/user.spec.js

```diff
@@ -0,0 +1,43 @@
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Pushan Mitra on 2019-03-07.
+//
+import assert from 'assert';
```

Comment:
I think it would be better to use the build in assertion to Jest because it will be the same between both Web and API. This just introduces another library that the next developer needs to support.